locales/cy/translations.json

```diff
@@ -173,9 +173,14 @@
             },
             "limitedPartner":{
                 "usualResidentialAddress":{
-                    "title": "WELSH - What is the limited partner's usual residential address",
+                    "title": "WELSH - What is the limited partner's usual residential address?",
```

Comment:
Simple change to add question mark, which was missing. In the English resource file too.